src/js/components/settings/organization/organization.js

```diff
@@ -143,7 +143,7 @@ export const Organization = ({
           }
         />
       </List>
-      {canPreview && isEnterprise && (
+      {(window.location.hostname.includes('staging') || canPreview) && isEnterprise && (
```

Comment:
one day we should have `isStaging`

Reply:
hm... true... but for now I'll refrain from changing this to keep the change to a minimum - there is #2994 waiting to get in first 🙈... feel free to ping me next week 😅

Reply:
next week you say? :)